### DISCUSSION TITLE
Solution file?

### DISCUSSION THREAD

**Discussion by JasonBSteele** [winleafs/Winleafs#197] [Q&A] [answered]
Hi, is the omission of a Solution file intentional?

**Comment by bartdebever** [comment, marked as answer]
Hey there, the solution file can be found in the Winleafs.WPF folder.

We should definitely move this to the root of the repository, apologies for the inconvenience.

**Comment by JasonBSteele** [comment]
Haha - so it is - I've just recreated it! Haven't had a chance to play with the app yet - but interested in reusing your hard work as an API for my tinkering. Thanks!
